addon/components/o-s-s/currency-input.ts

```diff
@@ -7,7 +7,8 @@ import { isEmpty } from '@ember/utils';
 interface OSSCurrencyInputArgs {
   currency: string;
   value: number;
-  onChange(currency: string, value: number): void;
+  onChange(currency: string | undefined, value: number): void;
```

Comment:
I think it's the `value` that's supposed to be undefined in case of `onlyCurrency` here